ui/apps/platform/src/Containers/Administration/Events/AdministrationEventsEmptyState.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactElement } from 'react';
+import { Bullseye, EmptyState, EmptyStateBody, EmptyStateIcon } from '@patternfly/react-core';
+import { FilterIcon } from '@patternfly/react-icons';
+import { Tbody, Td, Tr } from '@patternfly/react-table';
+
+export type AdministrationEventsEmptyStateProps = {
+    colSpan: number;
+    hasFilter: boolean;
+};
+
+function AdministrationEventsEmptyState({
+    colSpan,
+    hasFilter,
+}: AdministrationEventsEmptyStateProps): ReactElement {
+    return (
+        <Tbody>
+            <Tr>
+                <Td colSpan={colSpan}>
+                    <Bullseye>
+                        <EmptyState>
+                            {hasFilter && <EmptyStateIcon icon={FilterIcon} />}
+                            <EmptyStateBody>
+                                {hasFilter ? 'No events found' : 'No events'}
```

Comment:
nit: I think for the "No events found" case, it would be nice to add something like "Try expanding your search." (or something like that)

Doesn't need to be changed for this PR, though.